crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/OutgoingsMapper.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import uk.gov.justice.laa.crime.applications.adaptor.enums.HousingDetails;
+import uk.gov.justice.laa.crime.applications.adaptor.enums.OutgoingDetails;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.Outgoing;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class OutgoingsMapper {
+    private static final String HOUSING = "housing";
+    private static final String BOARD_LODGINGS_TYPE = "board_lodgings";
+    private static final String BOARD_LODGINGS = "Board lodgings";
+
+    public List<AssessmentDetail> mapOutgoings(List<Outgoing> outgoings, Object housingPaymentType) {
+        List<AssessmentDetail> assessmentDetails = new ArrayList<>();
+
+        if (Objects.nonNull(outgoings)) {
+            for (Outgoing outgoing : outgoings) {
+                AssessmentDetail assessmentDetail = new AssessmentDetail();
+                String outgoingsType = outgoing.getType().value();
+                // If the type is 'housing', we map against the housing codes instead
+                if (outgoingsType.equals(HOUSING) && Objects.nonNull(housingPaymentType)) {
```

Comment:
I would swap the order of this equality check to aid null safety
`outgoingsType.equals(HOUSING)`
becomes
`HOUSING.equals(outgoingsType)`